backend/src/main/java/eu/bbmri_eric/negotiator/governance/resource/dto/ResourceDTO.java

```diff
@@ -35,6 +35,12 @@ public class ResourceDTO {
   @Nullable
   private String name;
 
+  @Nullable private String description;
+
+  @Nullable private String contactEmail;
+
+  @Nullable private String uri;
+
```

Comment:
again this DTO is deprecated. The ResourceResponseModel and ResourceWithStatusDTO are used

Reply:
The problem here is that it has been deprecated, but several tests, changed in this PR, still use this class. Should we change again the tests and definitely remove this class ?

Reply:
do not remove the class for now, its fine that some tests use it